georocket-server/integration-tests/test.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
-set -e
```

Comment:
I think we should keep this file for local testing

Reply:
reverted